.github/workflows/deploy-dev.yml

```diff
@@ -108,6 +108,40 @@ jobs:
       AGOL_AREA_RESTRICTIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Area_Restrictions/FeatureServer/0/
       AGOL_BANS_AND_PROHIBITIONS: https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/test_British_Columbia_Bans_and_Prohibition_Areas/FeatureServer/0/
 
+      #public-mobile-war
```

Comment:
Input names can only contain alphanumeric containers, '-' or '_', not '.' characters. See https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_id